stress/stress_test.go

```diff
@@ -0,0 +1,129 @@
+package stress
+
+import (
+	//"encoding/json"
+	//"net/http"
+	//"net/http/httptest"
+	"testing"
+	"time"
+
+	//"github.com/influxdb/influxdb/client"
+)
+
+func TestTimer_StartTimer(t *testing.T) {
+	var epoch time.Time
```

Comment:
I feel there are too many blank lines here. Check out our existing unit tests, the style is much tighter.